pages/about.html

```diff
@@ -1,12 +1,31 @@
+<!-- Should include info about me, interests, background, etc -->
+<!-- Has <header>, <footer>, and <nav> -->
+<!-- <nav>, has links to all other pages -->
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About | Maria</title>
+  <link href="/Users/mobregong/ada/projects/personal-portfolio-site/styles/style.css" rel="stylesheet">
```

Comment:
This path should be a relative link 
```suggestion
  <link href="../styles/style.css" rel="stylesheet">
```